src/state/slices/portfolioSlice/utils.ts

```diff
@@ -270,13 +227,51 @@ export const accountToPortfolio: AccountToPortfolio = ({
         break
       }
       default:
-        break
+        assertUnreachable(chainNamespace)
     }
   })
 
   return portfolio
 }
 
+export const checkAccountHasActivity = (account: Account<ChainId>) => {
+  const { chainId } = account
+  const { chainNamespace } = fromChainId(chainId)
+
+  switch (chainNamespace) {
+    case CHAIN_NAMESPACE.Evm: {
+      const ethAccount = account as Account<KnownChainIds.EthereumMainnet>
+
+      const hasActivity =
+        bnOrZero(ethAccount.chainSpecific.nonce).gt(0) || bnOrZero(ethAccount.balance).gt(0)
```

Comment:
We probably don't need the latter as nonce will always be gt(0) if there's a balance but since this is such a cheap check, doesn't hurt to!